website/docs/advanced-concepts/access-control/granular-access-control/roles.md

```diff
@@ -0,0 +1,242 @@
+* **Groups and Roles:** These are resources related to the roles available for the workspaces in an Appsmith instance. For example, if you have two workspaces - HR and Finance - then roles related to HR and finance would be available.
+* **Others:** These are resources related to workspaces and audit logs in Appsmith.
+
+## Permissions
+
+ A permission refers to the ability to perform a certain operation on a resource. Multiple permissions can be clubbed under a role and a permission can be present in multiple roles. The table below lists all the permissions available in Appsmith -
+
+
+|  Permissions       |    Description                                                     |   Resource   |
+| ---------------    |------------------------------------------------------------------- | ------------ |
+| **Create**         | Enables permission to create, edit, view, and delete resources. You can also grant users access to create groups and roles. | [App Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |
+| **Edit**           |Enables permission to edit and view resources. You can also grant users access to create groups and roles | [App Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |
+| **View**           |Enables permission to only view resources. With the View permission, a user can only view the existing groups and roles | [App Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |
+| **Delete**         |Enables permission to delete resources. You can also grant users access to delete groups and roles |[App Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |  
+| **Export**         |Grants users permission to export applications in a workspace. | [App Resources](#application-resources-permissions) | 
+| **Invite**         |Grants users permission to invite users to a user group | [Groups and roles](#groups-and-roles-permissions)     |
+| **Make public**    |Grants users permission to make the applications public in a workspace |[App Resources](#application-resources-permissions)|
+| **Execute**        |Grants users permission to execute queries on a datasource | [Datasource and queries](#datasource-and-queries-permissions) |
+| **Remove users**   |Grants users permission to remove a user from a group| [Groups and roles](#groups-and-roles-permissions) |
+
```

Comment:
This table mentions everywhere `App Resources` and not `Application Resources`. Pointing it out in case we want it to match the tab name on UI.